src/components/MyObservations/SimpleErrorHeader.tsx

```diff
@@ -17,7 +17,12 @@ export interface Props {
 const SimpleErrorHeader = ( {
   isConnected
 }: Props ) => {
+  const { isDefaultMode } = useLayoutPrefs( );
   const { t } = useTranslation( );
+
+  if ( !isDefaultMode ) {
```

Comment:
From the designs, I think this is also supposed to be shown in advanced mode:
https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=23663-77654&t=pMuk9C20LY8bFrMb-4